web/docs/understand-vast/query-language/expressions.md

```diff
@@ -2,8 +2,201 @@
 sidebar_position: 0
 ---
 
+import MissingDocumentation from '@site/presets/MissingDocumentation.md';
+
 # Expressions
 
-import MissingDocumentation from '@site/presets/MissingDocumentation.md';
+VAST's expression language makes it easy to describe a relevant subset of
+interest.
+
+An *expression* is a constraint over an event that evaluates to `true` or
```

Comment:
I'm reluctant with this change because we actually use the term "predicate" for something else.

Reply:
Fair enough. I think I'm just not happy with 

> An *expression* is a constraint over an event

because for me it just doesn't flow. I can see many ways:

> An expression is a constraint over events

> An expression filters events...


We (me) may also be overthinking this - it's a single word / formulation, that - even if someone with my taste reads it - doesn't make the docs  bit less accessible or understandable.

Reply:
You're not overthinking. If you asked yourself this, what would our users do? 😱

Maybe s/constraint/function/?